src/main/kotlin/com/asap/asapbackend/domain/announcement/presentation/SchoolAnnouncementController.kt

```diff
@@ -0,0 +1,94 @@
+package com.asap.asapbackend.domain.announcement.presentation
+
+import com.asap.asapbackend.domain.announcement.application.SchoolAnnouncementService
+import com.asap.asapbackend.domain.announcement.application.dto.*
+import io.github.oshai.kotlinlogging.KotlinLogging
+import org.springframework.web.bind.annotation.*
+
+private val logger = KotlinLogging.logger {}
+
+@RestController
+class SchoolAnnouncementController(
+    private val announcementService: SchoolAnnouncementService
+) {
+
+
+    @GetMapping(AnnouncementApi.V1.SCHOOL_ANNOUNCEMENT)
+    fun getSchoolAnnouncement(
+        @RequestParam size: Int,
+        @RequestParam page: Int
+    ): GetSchoolAnnouncementSlice.Response {
+        return announcementService.getSchoolAnnouncementSlice(GetSchoolAnnouncementSlice.Request(size, page))
+    }
+
+    @GetMapping(AnnouncementApi.V1.SCHOOL_ANNOUNCEMENT_DETAIL)
+    fun getSchoolAnnouncementDetail(
+        @PathVariable schoolAnnouncementId: Long
+    ): GetSchoolAnnouncementDetail.Response {
+        return announcementService.getSchoolAnnouncementDetail(GetSchoolAnnouncementDetail.Request(schoolAnnouncementId))
+    }
```

Comment:
인자가 한개 뿐이면
```suggestion
        return announcementService.getSchoolAnnouncementDetail(schoolAnnouncementId)
    }
```
이런식으로 넘겨 줘도 괜찮지 않아? Request는 항상 DTO에 넣어서 주는게 좋다고 생각해?? 어떻게 생각해

Reply:
밑에는 schoolAnnouncementId request에 안넣고 보냈던데 기준이 뭐야??

Reply:
바디와 path variable이 있을 경우에는 같이 묶어서 보낼 수 없어서 그냥 넘겨줬어.

Reply:
요청을 보낼때 그냥 필드를 통해서 넘겨줄 수 있지만. 만일 api 스팩이 바뀔때 저렇게 필드를 바로 넣으면 변경에 있어서 어렵게 만들어서 Request 인스턴스를 만들고 넘겼어